project/Build.scala

```diff
@@ -465,13 +468,18 @@ object Build {
 
       test in Test := {
         // Exclude legacy tests by default
+        (testOnly in Test).toTask(" -- --exclude-categories=java.lang.Exception,dotty.tools.dotc.SlowTests").value
+      },
+
+      testAll in Test := {
+        // Exclude legacy tests by default
         (testOnly in Test).toTask(" -- --exclude-categories=java.lang.Exception").value
```

Comment:
While you're at it, could you create a meaningful category for legacy tests?

Reply:
I will